songs.js

```diff
@@ -0,0 +1,35 @@
+var songs = [];
+
+songs[songs.length] = "Legs > by Z*ZTop on the album Eliminator";
+songs[songs.length] = "The Logical Song > by Supertr@amp on the album Breakfast in America";
+songs[songs.length] = "Another Brick in the Wall > by Pink Floyd on the album The Wall";
+songs[songs.length] = "Welco(me to the Jungle > by Guns & Roses on the album Appetite for Destruction";
+songs[songs.length] = "Ironi!c > by Alanis Moris*ette on the album Jagged Little Pill";
+
+function addMusic() {
+}
+
+var allSongs = songs.push(songs[songs.length]);
```

Comment:
I don't think you need this array function ... let's console songs and see if it is already an array.